src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+import { FC } from 'react';
+
+export const HomePage:FC = () => (
```

Comment:
add spaces in such cases
```suggestion
export const HomePage: FC = () => (
```